src/binaryen-c.h

```diff
@@ -1676,10 +1678,10 @@ BINARYEN_API void BinaryenMemoryFillSetSize(BinaryenExpressionRef expr,
 
 // Gets the value expression tested to be null of a `ref.is_null` expression.
 BINARYEN_API BinaryenExpressionRef
-BinaryenRefIsNullGetValue(BinaryenExpressionRef expr);
+BinaryenRefIsGetValue(BinaryenExpressionRef expr);
 // Sets the value expression tested to be null of a `ref.is_null` expression.
-BINARYEN_API void BinaryenRefIsNullSetValue(BinaryenExpressionRef expr,
-                                            BinaryenExpressionRef valueExpr);
+BINARYEN_API void BinaryenRefIsSetValue(BinaryenExpressionRef expr,
+                                        BinaryenExpressionRef valueExpr);
```

Comment:
We probably need new `GetOp` and `SetOp` functions, too.

Reply:
Right, yes. That wasn't blocking tests from passing, so I'm deferring it. My main priority is to get end-to-end support of GC prototype 3 finished, to unblock commandline tool usage first.